app/src/main/java/gov/hhs/cdc/trustedintermediary/external/reportstream/ReportStreamConnection.java

```diff
@@ -0,0 +1,84 @@
+package gov.hhs.cdc.trustedintermediary.external.reportstream;
+
+import gov.hhs.cdc.trustedintermediary.wrappers.*;
+import java.io.IOException;
+import java.util.Map;
+import javax.inject.Inject;
+import org.jetbrains.annotations.NotNull;
+
+public class ReportStreamConnection implements ClientConnection {
```

Comment:
What do you think about removing this level of abstraction and put most all of this logic into `ReportStreamLabOrderSender`?

Reply:
Sounds good.